Headers/DebugServer2/Host/Linux/PTrace.h

```diff
@@ -66,6 +66,7 @@ class PTrace : public POSIX::PTrace {
 
 public:
   ErrorCode getSigInfo(ProcessThreadId const &ptid, siginfo_t &si) override;
+  ErrorCode getEventMessage(ProcessId pid, unsigned long &data);
```

Comment:
I think the type of `pid` should be `ProcessThreadId` and `getEventMessage()` should convert it to a `pid_t` using `CHK(ptidToPid(ptid, pid));` (see the implementation of `PTrace::getSigInfo()`).